build_projects/dotnet-host-build/StubPackageBuilder.cs

```diff
@@ -42,9 +42,8 @@ private void BuildStubBits(DotNetCli dotnet, string intermediateDirectory)
         {
             var projectJson = new StringBuilder();
             projectJson.Append("{");
-            projectJson.Append("  \"dependencies\": { \"NETStandard.Library\": \"1.5.0-rc2-24027\" },");
-            projectJson.Append("  \"frameworks\": { \"netcoreapp1.0\": { \"imports\": [\"netstandard1.5\", \"dnxcore50\"] } },");
-            projectJson.Append("  \"runtimes\": { \"win7-x64\": { } },");
```

Comment:
If it works then it doesn't matter, the rid here is meaningless with respect to the actual platform though. The entire intention is to produce a stub package with the current rid in the *name*. The contents of those packages don't matter.